pypeit/scripts/trace_edges.py

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env python
```

Comment:
Thanks this script is great!

Reply:
PS -- But I've never actually tested it on tracing a single image...